pygtc/pygtc.py

```diff
@@ -1258,11 +1274,17 @@ def __plot2d(ax, nChains, chains2d, weights, nBins, smoothingKernel,
     for k in range(nChains):
         if plotData[nChains-1-k] is not None:
             for l in range(nContourLevels):
-                ax.contour(plotData[nChains-1-k],
+                if contourLinewidth is no None:
```

Comment:
For starters, "no" is not a valid python test. It's clear you have not tested this code (because this code would immediately break). Please do not issue a PR with untested code in the future.

In any case, my suggestion here is the same as in the plot1d method. `linewidths=contourLinewidth or 1`. If you don't like doing that in a function call, then you can rewrite the variable at the top of the method with `countourLinewidth=contourLinewidth or 1`. This way the next person that needs to add a parameter doesn't need to deal with a whole bunch of branching if/else tests.